README.md

```diff
@@ -1 +1,111 @@
 # AppMap Gradle plugin
+
+- [AppMap Gradle plugin](#appmap-gradle-plugin)
+    - [Quickstart](#quickstart)
+    - [About](#about)
+    - [Plugin tasks](#plugin-tasks)
+    - [Plugin configuration](#plugin-configuration)
+    - [Troubleshooting](#troubleshooting)
+
+## Quickstart
+
+First, ensure you have a
+[properly configured `appmap.yml`](https://github.com/applandinc/appmap-java#configuration)
+in your root project directory. A basic configuration may look like:
+
+```yml
+# appmap.yml
+name: my_organization/my_application
+
+packages:
+  # List the packages you'd like to record here.
+  - path: com.myorganization.myapplication
+
+  # Individual classes or methods can be chosen as well.
+  #
+  # - path: com.myorganization.myapplication.MyClass
+  # - path: com.myorganization.myapplication.MyClass#myInstanceMethod
+  # - path: com.myorganization.myapplication.MyClass.myStaticMethod
+  #
+  # Optionally, include paths of packages, classes
+  # or methods that you'd like to exclude from recording
+  #
+  # - exclude: com.myorganization.myapplication.util
+  # - exclude: com.myorganization.myapplication.MyClass
+  # - exclude: com.myorganization.myapplication.MyClass#myInstanceMethod
+  # - exclude: com.myorganization.myapplication.MyClass.myStaticMethod
+```
+
+Next, add the following plugin definition to your `build.gradle`:
+```groovy
+// Apply the plugin
+        apply plugin: 'com.appland.appmap'
+// Set up AppMap agent, default parameters 
+        appmap {
+            configFile = file("$projectDir/appmap.yml")
+            outputDirectory = file("$projectDir/build/appmap")
+            skip = false
+            debug = "info"
+            debugFile = "build/appmap/agent.log"
+            eventValueSize = 1024
```

Comment:
We should use the $buildDir variable instead of hardcoded path wherever appropriate.

Reply:
done